api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java

```diff
@@ -80,13 +80,15 @@ public static EncounterDiagnosesByObsTagHandler setupEncounterDiagnosesByObsTagH
     /**
 	 * @see ModuleActivator#willRefreshContext()
 	 */
+    @Override
```

Comment:
At this point, you have 4 spaces instead of one tab.

Reply:
updated